torchvision/csrc/cpu/image/image_read_mode.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+/* Should be kept in-sync with Python ImageReadMode enum */
+using ImageReadMode = int64_t;
+#define IMAGE_READ_MODE_UNCHANGED 0
+#define IMAGE_READ_MODE_GRAY 1
+#define IMAGE_READ_MODE_GRAY_ALPHA 2
+#define IMAGE_READ_MODE_RGB 3
+#define IMAGE_READ_MODE_RGB_ALPHA 4
```

Comment:
nit: new line

I'll add it along with the other proposed corrections to minimize CI runs.